### ISSUE TITLE
Feat: support for remaining_accounts

### ISSUE THREAD

**Issue by magj2006** [solana-playground/solana-playground#296]
Hey, Is there a plan to support remaining_accounts ?

**Comment by acheroncrypto** [comment]
It's supported from TypeScript with the `remainingAccounts` method. I'm guessing this request is specifically about the test UI?

**Comment by magj2006** [comment]
Yeah, I think it will be very helpful and convenient while testing using UI.
